File: frontend_python_respaldo/test_conexion.py
from utils.conexion_mysql import obtener_conexion

conexion = obtener_conexion()

if conexion:
    print("===================================")
    print(" CONEXIÓN EXITOSA A MYSQL")
    print(" Base de datos: lpp_smart_erp")
    print("===================================")

    conexion.close()

else:
    print("No fue posible conectar.")

File: frontend_java/src/TestConexion.java
import java.sql.Connection;

public class TestConexion {

    public static void main(String[] args) {

        try (Connection conexion = ConexionDB.obtenerConexion()) {

            if (conexion != null) {
                System.out.println("CONEXIÓN EXITOSA A MYSQL");
                System.out.println("Base de datos: lpp_smart_erp");
            }

        } catch (Exception e) {
            System.out.println("ERROR DE CONEXIÓN:");
            e.printStackTrace();
        }
    }
}